docs/developer/extending/apps/architecture/app-permissions.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: App permissions
+---
+
+[Permissions](developer/permissions.mdx) are a way to control what operations can be performed by an app.
+
+## Managing permissions
+
+When you install an app, Saleor grants it a set of permissions. The initial app permissions are drawn from [the app's manifest file](developer/extending/apps/architecture/manifest.mdx) and stored in the [`App`](api-reference/apps/objects/app.mdx) object under the `permissions` field.
+
+Permissions can also be modified after the app is installed. You can change them using the [appUpdate](api-reference/apps/mutations/app-update.mdx) mutation:
+
+```graphql
+mutation UpdateAppPermissions($id: ID!) {
+  appUpdate(id: $id, input: { permissions: [MANAGE_ORDERS] }) {
+    app {
+      permissions {
+        name
+        code
+      }
+    }
+  }
+}
+```
+
+## Using the app
+
+Apps are only visible for the users who have the `MANAGE_APPS` permission.
+
+:::info
+
+To install, deactivate or delete an app, the user must have the `MANAGE_APPS` permission, as well as all the app permissions.
+
+:::
+
+Let's consider an example with the Orders App, which sends orders to an external service, to differentiate between users who can use the app and those who can manage it. The app requires the `MANAGE_ORDERS` permission.
+
+A user who only has the `MANAGE_APPS` permission can open the installed app, but they won't be able to deactivate or delete it.
```

Comment:
Is it correct? I thought if you have MANAGE_APPS you can remove and disable it too

Reply:
yes